_source/_posts/2024-02-29-3pc.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
The slug contributes to SEO I believe via the keywords. I'd at least change the slug to 
`2024-02-29-third-party-cookies`

Also try to work in third party cookies into more headers to improve SEO